minecraft_servers/porgverse/manifest.json

```diff
@@ -12,6 +12,12 @@
   ],
   "location": {
     "country": "The Netherlands",
+    "city": "Amsterdam",
     "country_code": "NL"
-  }
+  },
+  "discord": {
+    "server_id": 811987858402574366,
+    "rename_to_minecraft_name": true
+  },
+  "user_stats": "https://web.porgverse.nl/player/{userName}"
```

Comment:
Website is not available

![Bildschirmfoto 2021-09-03 um 23 07 24](https://user-images.githubusercontent.com/45363287/132065386-c177e1c0-2b9c-427c-a9f2-78c1e673f325.png)

Reply:
Only available to known players, try my username: Jazzkuh
https://web.porgverse.nl/player/Jazzkuh

Reply:
Ah, I see, okay! Could you make this more clear in the future? Because every user can see the stats button on the LABY.net page with the username. When the user is not a known player, this will be a little bit confusing.